community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
I still don't see it. Points 3 and 4 are very different from other points. Points 1 and 2 just say where to ask for help (which will be obvious to most). But points 3 and 4 actually give information not given elsewhere and is not obvious. Point 5 seems to be a condition on getting offline activities accepted for KPI.

I am not saying any point is less important than others but all we are doing here is place all miscellaneous items in a common bucket. We should put things in their respective sections as much as possible.